docs/opcodes/30.mdx

```diff
@@ -3,22 +3,23 @@ fork: Frontier
 group: Environmental Information
 ---
 
-*Index 1 is top of the stack. See [PUSH](/#60).*
+_Index 1 is top of the stack. See [PUSH](/#60)._
 
 ## Stack output
 
 0. `address`: the 20-byte address of the current account.
 
 ## Example
 
-| * | Input | Output |
-|--:|------:|-------:|
-| `1` | | `0x9bbfed6889322e016e0a02ee459d306fc19545d8` |
+|  \* | Input |                                       Output |
+| --: | ----: | -------------------------------------------: |
```

Comment:
There are a lot of changes in these `.mdx` files, those seem to be only code style changes and don't have any real visual impact, right?

Reply:
These changes and because of the prettier in the whole code, but if you prefer I can isolate that.

Reply:
No if prettier fixed those, that's fine, let's keep them. The must have not been done properly in the first place.